Description: Set default cutchars, fix double-click word selection
 This patch appeared in 9.05-2. 
 .
 It was unapplied in 9.10-1 at upstream's request to avoid deviation with
 upstream urxvt's behavior.
Author: Decklin Foster <decklin@red-bean.com>
Origin: vendor
Bug-Debian: http://bugs.debian.org/377194
Reviewed-By: Unapplied by Ryan Kavanagh <ryanakca@kubuntu.org>
Last-Update: 2011-03-06

--- rxvt-unicode-9.09.orig/src/perl/selection
+++ rxvt-unicode-9.09/src/perl/selection
@@ -12,11 +12,13 @@ sub on_user_command {
 sub on_init {
    my ($self) = @_;
 
+   my $cutchars = '\\`"\'&()*,;<=>?@[]{|}';
    if (defined (my $res = $self->resource ("cutchars"))) {
-      $res = $self->locale_decode ($res);
-      push @{ $self->{patterns} }, qr{\G [\Q$res\E[:space:]]* ([^\Q$res\E[:space:]]+) }x;
+      $cutchars = $self->locale_decode ($res);
    }
 
+   push @{ $self->{patterns} }, qr{\G [\Q$cutchars\E[:space:]]* ([^\Q$cutchars\E[:space:]]+) }x;
+
    for (my $idx = 0; defined (my $res = $self->x_resource ("selection.pattern-$idx")); $idx++) {
       $res = $self->locale_decode ($res);
       utf8::encode $res;
